src/main/groovy/com/spidasoftware/schema/conversion/changeset/calc/TensionGroupRenameClientDataChangeSet.groovy

```diff
@@ -0,0 +1,48 @@
+/*
+ * ©2009-2020 SPIDAWEB LLC
+ */
+package com.spidasoftware.schema.conversion.changeset.calc
+
+
+import com.spidasoftware.schema.conversion.changeset.ChangeSet
+import com.spidasoftware.schema.conversion.changeset.ConversionException
+import groovy.transform.CompileStatic
+import groovy.util.logging.Log4j
+
+/**
+ * To avoid confusing users with the introduction of "Advance Wires"...
+ * Rename existing tension group names from...
+ * Advanced > Nonlinear Stress-Strain
+ * Dynamic > Elastic (Dynamic)
+ * Static > Static
+ */
+@Log4j
+@CompileStatic
+class TensionGroupRenameClientDataChangeSet extends ChangeSet {
+
+	void applyToClientData(Map clientDataJSON) throws ConversionException {
+		clientDataJSON?.wires?.each { Map wire ->
+			wire?.tensionGroups?.each { Map tensionGroup ->
+				String tensionGroupName = tensionGroup?.name
+				if (tensionGroupName.toLowerCase() == "advanced") {
+					tensionGroup?.name = "Nonlinear Stress-Strain"
+				} else if (tensionGroupName.toLowerCase() == "dynamic") {
+					tensionGroup?.name = "Elastic (Dynamic)"
+				}
+			}
+		}
+	}
+
+	void revertClientData(Map clientDataJSON) throws ConversionException {
+		clientDataJSON?.wires?.each { Map wire ->
+			wire?.tensionGroups?.each { Map tensionGroup ->
+				String tensionGroupName = tensionGroup?.name
+				if (tensionGroupName.toLowerCase() == "nonlinear stress-strain") {
```

Comment:
rename dynamic in the changeset but not advanced. Advanced was introduced in 7.3.